code/modules/mob/living/brain/MMI.dm

```diff
@@ -28,6 +29,15 @@
 	var/next_possible_ghost_ping
 	//Used by syndie MMIs, stores the master's mind UID for later referencing
 	var/master_uid = null
+	/// Extended description on examine_more
+	var/extended_desc = "Development of Man-Machine Interfaces can be dated all the way back to the late 20th century on Earth, but the first viable designs didn't emerge until 2408 when Zeng-Hu Pharmaceuticals and Bishop Cybernetics unveiled a co-developed unit and established dominance over the niche market that persists into the present day. \
```

Comment:
```suggestion
	var/extended_desc = "Development of Man-Machine Interfaces can be dated all the way back to the late 20th century within the Sol system, but the first viable designs didn't emerge until 2408 when Zeng-Hu Pharmaceuticals and Bishop Cybernetics unveiled a co-developed unit and established dominance over the niche market that persists into the present day. \
```
Directly naming earth feels a bit strange to me. could be worth seeing in the lore chat if it'd be better to stay as earth, or sol system though